lib/tasks/orders_packages.rake

```diff
@@ -0,0 +1,16 @@
+namespace :goodcity do
+  namespace :orders_packages do
+    desc <<-DESC
+      Initializes the dispatched_quantity field of orders_packages.
+      If the orders_package is marked as dispatched, sets the `dispatched_quantity` field to the quantity designated
+    DESC
+    task init_dispatched_quantity: :environment do
+      count = 0
+      OrdersPackage.dispatched.find_each do |orders_package|
+        orders_package.update(dispatched_quantity: orders_package.quantity)
+        count += 1
+      end
+      puts "Task completed. #{count} records modified"
+    end
```

Comment:
This is fine, Can also run a direct SQL query inside the migration to make the upgrade smooth. I'll put it on there as a comment for you to decide..

Reply:
Oh definitely, I've been aching to do that. I just try to fit the standards